src/meta/proto-conv/tests/it/user_stage.rs

```diff
@@ -53,6 +53,50 @@ fn test_user_stage_oss_latest() -> anyhow::Result<()> {
     Ok(())
 }
 
+#[test]
+fn test_user_stage_fs_v25() -> anyhow::Result<()> {
+    let user_stage_info_v25 = vec![
+        10, 17, 102, 115, 58, 47, 47, 100, 105, 114, 47, 116, 111, 47, 102, 105, 108, 101, 115, 26,
```

Comment:
Consider adopting(copy-and-paste) the latest compatibility test style: 
 https://github.com/datafuselabs/databend/blob/78d127257519bea475ad1361ee0359da764a443b/src/meta/proto-conv/tests/it/v024_table_meta.rs

When getting rid of versions that are no longer supported in the future, it will be easier to do so.